main.tf

```diff
@@ -1,18 +1,11 @@
-########################################################################################################################
-# Input variable validation
-# (approach based on https://github.com/hashicorp/terraform/issues/25609#issuecomment-1057614400)
-#
-# TODO: Replace with terraform cross variable validation: https://github.ibm.com/GoldenEye/issues/issues/10836
-########################################################################################################################
-
 ########################################################################################################################
 # Locals
 ########################################################################################################################
 
 locals {
```

Comment:
check Redis
`# If no value passed for 'backup_encryption_key_crn' use the value of 'kms_key_crn' and perform validation of 'kms_key_crn' to check if region is supported by backup encryption key.
` comment is added